scss/bootstrap/overrides/style/_navs.scss

```diff
@@ -269,3 +199,76 @@
     }
   }
 }
+
+
+//== Nav tabs in a panel-default
+.panel-default {
+
+  // Horizontal nav tabs
+  .nav-tabs:not(.nav-tabs-stacked) {
+    $padding-x: nth($crm-panel-subheading-padding, 2);
```

Comment:
Why use a variable if we can use it directly?
`padding: 5px 0 0 nth($crm-panel-subheading-padding, 2);